### ISSUE TITLE
'instalooter' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by InfinitiStudios** [nathan-149/automated_youtube_channel#59]
I tried to run this command in cmd, but its apperantly "'instalooter' is not recognized as an internal or external command,
operable program or batch file." i have no idea what to do so if someone knows pls say, thanks

**Comment by HeikaGrant** [comment]
Yeah same here, I also downloaded the git repo but still could not get the command to be recognized.

**Comment by mhaneman** [comment]
Restarting my PC seemed to do the trick for me. However, this doesn't matter because instalooter is depreciated. I'm gonna fork the repo and see if I can remove replace instalooter instances with instaloader

**Comment by cris-i** [comment]
How's it going mhaneman? I tried changing the python code so it downloaded videos from specific accounts that I would determine but it doesn't seem to work. I guess I need much more knowledge and practice to get it done
